modules/administration/pages/ssl-certs-selfsigned.adoc

```diff
@@ -17,63 +24,13 @@ The host name of the SSL keys and certificates must match the fully qualified ho
 You can set the host names in the ``X509v3 Subject Alternative Name`` section of the certificate.
 You can also list multiple host names if your environment requires it.
```

Comment:
This apply only to Import Certificate. Here we generate everything on our own. This can be removed